### DISCUSSION TITLE
Create UE with programmable SIM

### DISCUSSION THREAD

**Discussion by sruthi81** [srsran/srsRAN_Project#929] [General Help]
Respected all,
Here I am asking a general help from your side to connect my UE with srsRAN gnb via USRP X310. I have no idea about how the programmable sim connect with core network practically. I am here by adding my UE details. Could you please suggest a good programmable SIM.
Samsung Galaxy M33 5G
Core Network : Open5Gs
gnb: srsRAN

Thanks
Sruthi

**Comment by robertfalkenberg** [comment]
We had good experience with [SysmoISIM-SJA2](https://sysmocom.de/products/sim/sysmousim/index.html)
as mentioned here: https://docs.srsran.com/projects/project/en/latest/tutorials/source/cotsUE/source/index.html

The successor should work in the same way.
